### PR TITLE
Update Temurin Spec | Linux

### PR DESCRIPTION
**Pull request by chandrashritii**
*Description* 

Update Temurin spec | Linux

This PR fixes https://github.com/adoptium/installer/issues/417

Signed-off-by: Shriti Chandra [shchandr@redhat.com](mailto:shchandr@redhat.com)

### CODE REVIEW

**Comment by karianna** [comment]
@chandrashritii you'll need to sign the Eclipse Foundation CLA

**Review by jiekang — CHANGES_REQUESTED** [review]
Thanks Shriti!

Please also increment the `spec_release` field [1] and add a changelog entry for the spec files. Changes that alter the result needs an incremented `spec_release` so users can upgrade to it properly.

The changelog entry can be a short description of the fix you're making, for example 'Fix alternatives linking'



[1]
https://github.com/adoptium/installer/blob/19f2f00c8655a3db4707f8b418fa621e6c1db002/linux/jdk/redhat/src/main/packaging/temurin/11/temurin-11-jdk.spec#L8

[2]
https://github.com/adoptium/installer/blob/19f2f00c8655a3db4707f8b418fa621e6c1db002/linux/jdk/redhat/src/main/packaging/temurin/11/temurin-11-jdk.spec#L265

**Comment by chandrashritii** [comment]
Thanks for the review! @jiekang Added the necessary changelogs

**Review by jiekang — CHANGES_REQUESTED** [review]

File: linux/jdk/redhat/src/main/packaging/temurin/17/temurin-17-jdk.spec [hunk 1]
```diff
@@ -259,6 +259,8 @@ fi
 /usr/lib/tmpfiles.d/%{name}.conf
 
 %changelog
+* Thu March 2 2023 Eclipse Adoptium Package Maintainers <temurin-dev@eclipse.org> 11.0.18.0.0.10.adopt0
```
[the remark this answers is not in the document]
Comment (jiekang) [reply linux/jdk/redhat/src/main/packaging/temurin/17/temurin-17-jdk.spec:262]: I think it's fine to remove 'Draft' status after confirming ECA and these last changes.

**Review by jiekang — APPROVED** [review]

**Comment by jiekang** [comment]
@chandrashritii There are still two commits with an unknown author/email address. You will need to modify those for the ECA check to pass. I recommend using `git rebase -i` and making this a single commit with your changes on top of the latest `adoptium:master` commit.

**Comment by jiekang** [comment]
Specifically:

https://github.com/adoptium/installer/pull/623/commits/eb2dac66d5b14531339d974fec6da75d0050ffea
https://github.com/adoptium/installer/pull/623/commits/3614035b76532236ec45bd2cb64101e8b8ad5e4e

**Comment by karianna** [comment]
https://stackoverflow.com/questions/34850831/change-git-email-for-previous-commits may help

**Comment by gdams** [comment]
This will also need updating to include the recently added JDK20 config files

**Comment by karianna** [comment]
@chandrashritii will need a rebase

**Review by karianna — APPROVED** [review]

**Comment by karianna** [comment]
@chandrashritii getting close, just some errors in the changelog format, see the failing GH Actions

**Comment by chandrashritii** [comment]
Thanks! @karianna Fixed the changelog format

File: linux/jdk/redhat/src/main/packaging/temurin/11/temurin-11-jdk.spec [hunk 2]
```diff
@@ -5,7 +5,15 @@
 #  $ rpmdev-vercmp 11.0.13.0.1___7 11.0.13.0.0+8
 #  11.0.13.0.0___8 == 11.0.13.0.0+8
 %global spec_version 11.0.18.0.0.10
+<<<<<<< HEAD
```
Comment (jiekang) [inline linux/jdk/redhat/src/main/packaging/temurin/11/temurin-11-jdk.spec:8]: @chandrashritii Looks like the merges need to be resolved here and in a few of the other files.

**Comment by karianna** [comment]
SUSE jobs still failing but no clear reason as to why

**Comment by jiekang** [comment]
I think the jdk 19 jobs fail due to the ExclusiveArch check in build.sh not accounting for specs without that field. However we can drop the changes to the 19 specs as 20 is out and ignore them.

**Comment by karianna** [comment]
@jiekang Will need a rebase now sorry :-(

**Comment by karianna** [comment]
Still have failing SUSE jobs...

**Review by steelhead31 — APPROVED** [review]
LGTM

**Pull request merged**